time/src/date_time.rs

```diff
@@ -114,9 +115,10 @@ impl MaybeOffset for offset_kind::Fixed {
     type OffsetType = UtcOffset;
     type OffsetKind = Self;
 
-    #[allow(clippy::undocumented_unsafe_blocks)] // false positive
-    // Safety: `UtcOffset` is the offset type.
-    const HAS_OFFSET: UnsafeBool = unsafe { UnsafeBool::new(true) };
+    const HAS_OFFSET: UnsafeBool = {
+        // Safety: `UtcOffset` is the offset type.
+        unsafe { UnsafeBool::new(true) }
+    };
```

Comment:
I don't care for this change. The existing code is acceptable, and it's just a bug with clippy.